### PR TITLE
Release v1.2.3

### PR DESCRIPTION
**Pull request by nimid**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [1.2.3] 2016-08-30
+- *`Added`* Add a new feature, localization
```
Comment (guzzilar) [inline CHANGELOG.md]: @nimid can you add `localisation  (JA)`

File: readme.txt [hunk 2]
```diff
@@ -30,6 +30,13 @@ From there:
 3. Omise Payment Gateway Checkout Form
 
 == Changelog ==
+= 1.2.3 =
+(Added) Add a new feature, localization
```
Comment (guzzilar) [inline readme.txt]: @nimid same here https://github.com/omise/omise-woocommerce/pull/30/commits/dce694da784ec3d875ba70c7a89e3b8bd3ebf7c1#r77788872

File: readme.txt [hunk 3]
```diff
@@ -58,6 +65,13 @@ Added Omise-Version into the cURL request header.
 Adds support for 3-D Secure feature
 
 == Upgrade Notice ==
+= 1.2.3 =
+(Added) Add a new feature, localization
```
Comment (guzzilar) [inline readme.txt]: @nimid same here https://github.com/omise/omise-woocommerce/pull/30/commits/dce694da784ec3d875ba70c7a89e3b8bd3ebf7c1#r77788872

File: languages/omise-woocommerce-ja.po [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+msgstr "決済エラー"
+
+#: includes/classes/class-omise-transfers-table.php:60
+msgid "Transfer Id"
+msgstr "振込ID"
+
+#: includes/classes/class-omise-transfers-table.php:61
+msgctxt "Transfer table column header"
+msgid "Sent"
+msgstr "完了"
```
Comment (guzzilar) [inline languages/omise-woocommerce-ja.po]: from @ryoko-omise suggested,

@nimid please change this translation to `振込要求済`

File: languages/omise-woocommerce-ja.po [hunk 5]
```diff
@@ -0,0 +1,500 @@
...
+
+#: includes/classes/class-omise-charges-table.php:60
+msgid "Charge Id"
+msgstr "課金ID"
+
+#: includes/classes/class-omise-charges-table.php:61
+msgid "Authorized"
+msgstr "認証済"
+
+#: includes/classes/class-omise-charges-table.php:62
```
Comment (guzzilar) [inline languages/omise-woocommerce-ja.po]: @nimid could you change this line from 
`'chrg_paid'       => Omise_Util::translate( 'Paid' ),`
to
`'chrg_paid'       => Omise_Util::translate( 'Captured' ),`

and this translation (for `'chrg_paid'       => Omise_Util::translate( 'Captured' ),`)
is `実売上化済`

File: languages/omise-woocommerce-ja.po [hunk 6]
```diff
@@ -0,0 +1,500 @@
...
+#: includes/classes/class-omise-transfers-table.php:62
+msgid "Paid"
+msgstr "振込済み"
```
Comment (guzzilar) [inline languages/omise-woocommerce-ja.po]: @nimid please change to `振込要求済`

File: languages/omise-woocommerce-ja.po [hunk 7]
```diff
@@ -0,0 +1,500 @@
...
+
+#: includes/classes/class-omise-charges-table.php:63
+#: includes/classes/class-omise-transfers-table.php:63
+msgid "Failure Message"
+msgstr "エラーメッセージ"
+
+#: includes/classes/class-omise-charges-table.php:64
+#: includes/classes/class-omise-transfers-table.php:64
+msgid "Created"
+msgstr "作成済"
```
Comment (guzzilar) [inline languages/omise-woocommerce-ja.po]: @nimid this one is `作成日`

File: languages/omise-woocommerce-ja.po [hunk 8]
```diff
@@ -0,0 +1,500 @@
...
+
+#: includes/omise-wc-setting.php:45
+msgid "Supported card icons"
+msgstr "対応するカードのアイコン"
+
+#: includes/omise-wc-setting.php:74
+msgid ""
+"This only controls the icons displayed on the checkout page.<br />It is not "
+"related to card processing on Omise payment gateway."
+msgstr "チェックアウトページのアイコンを設定のみ。"
```
Comment (guzzilar) [inline languages/omise-woocommerce-ja.po]: @nimid please change to 
```
この設定はチェックアウトページのアイコンのみに反映されます。
各カードブランドの決済対応に関する条件はOmiseと各加盟店との利用規約に基づきます。
```

File: languages/omise-woocommerce-ja.po (hunk 4 above)
Comment (nimid) [inline languages/omise-woocommerce-ja.po]: @guzzilar I found the transfer has sent and transfer has paid use the same Japanese word, 振込要求済.

Is this correct for the Japanese?

The screenshot below shows the sample of transfers history page.
screenshot-192 168 99 100 8080 2016-09-07 20-50-43

Comment (guzzilar) [inline languages/omise-woocommerce-ja.po]: @nimid nice catch!!
Sry, my fault

```
Transfer list page
SENT = 振込要求済　
PAID = 振込済
-------------------
```

**Comment by guzzilar** [comment]
👍

**Pull request merged**